cmd/cloud/server.go

```diff
@@ -298,9 +302,11 @@ var serverCmd = &cobra.Command{
 			"dev-mode":                               devMode,
 			"deploy-mysql-operator":                  deployMySQLOperator,
 			"deploy-minio-operator":                  deployMinioOperator,
+			"ndots-value":                            ndotsDefaultValue,
```

Comment:
left over local value from your other PR?

Reply:
Nope forgotten to add it on logger on previous pr. This fixes the initial absence.